non-free.md

```diff
@@ -51,6 +51,7 @@
 
 ## Project Management
   * [Active Collab](https://www.activecollab.com/) - Project management  - `⊘ Proprietary` `PHP`
+  * [Duet](https://duetapp.com/) `⊘ Proprietary`  - Invoicing and project management with an integrated client portal   ([Demo](https://duetapp.com/demo/#dashboard)) `PHP`
```

Comment:
Should be 

 * [Duet](https://duetapp.com/) - Invoicing and project management with an integrated client portal   ([Demo](https://duetapp.com/demo/#dashboard))  `⊘ Proprietary` `PHP`